Index: /@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
===================================================================
--- /@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ /@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -20,8 +20,9 @@
   cancelAnimation,
   useWorkletCallback,
   WithSpringConfig,
   WithTimingConfig,
+  ReduceMotion,
 } from 'react-native-reanimated';
 import { State } from 'react-native-gesture-handler';
 import {
   useScrollable,
@@ -97,9 +98,9 @@
 
     //#region extract props
     const {
       // animations configurations
-      animationConfigs: _providedAnimationConfigs,
+      animationConfigs,
 
       // configurations
       index: _providedIndex = 0,
       snapPoints: _providedSnapPoints,
@@ -171,8 +172,23 @@
         _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
     } = props;
     //#endregion
 
+      //#region animations configurations
+      const _providedAnimationConfigs = useMemo(() => {
+          if (!animationConfigs) {
+              return undefined;
+          }
+
+          if (ReduceMotion) {
+              // @ts-expect-error Property 'reduceMotion' does not exist on type 'WithSpringConfig | WithTimingConfig'.
+              animationConfigs.reduceMotion = ReduceMotion.Never;
+          }
+
+          return animationConfigs;
+      }, [animationConfigs]);
+      //#endregion
+
     //#region layout variables
     /**
      * This variable is consider an internal variable,
      * that will be used conditionally in `animatedContainerHeight`
@@ -721,8 +737,13 @@
         /**
          * force animation configs from parameters, if provided
          */
         if (configs !== undefined) {
+
+          if (ReduceMotion) {
+              configs.reduceMotion = ReduceMotion.Never;
+          }
+
           animatedPosition.value = animate({
             point: position,
             configs,
             velocity,
Index: /@gorhom/bottom-sheet/src/constants.ts
===================================================================
--- /@gorhom/bottom-sheet/src/constants.ts
+++ /@gorhom/bottom-sheet/src/constants.ts
@@ -1,6 +1,6 @@
 import { Dimensions, Platform } from 'react-native';
-import Animated, { Easing } from 'react-native-reanimated';
+import Animated, { Easing, ReduceMotion } from 'react-native-reanimated';
 
 const { height: WINDOW_HEIGHT, width: WINDOW_WIDTH } = Dimensions.get('window');
 const { height: SCREEN_HEIGHT, width: SCREEN_WIDTH } = Dimensions.get('screen');
 
@@ -71,13 +71,15 @@
   mass: 3,
   overshootClamping: true,
   restDisplacementThreshold: 10,
   restSpeedThreshold: 10,
+  ...(ReduceMotion ? { reduceMotion: ReduceMotion.Never } : {}),
 };
 
 const ANIMATION_CONFIGS_ANDROID = {
   duration: ANIMATION_DURATION,
   easing: ANIMATION_EASING,
+  ...(ReduceMotion ? { reduceMotion: ReduceMotion.Never } : {}),
 };
 
 const ANIMATION_CONFIGS =
   Platform.OS === 'ios' ? ANIMATION_CONFIGS_IOS : ANIMATION_CONFIGS_ANDROID;
Index: /@gorhom/bottom-sheet/src/utilities/animate.ts
===================================================================
--- /@gorhom/bottom-sheet/src/utilities/animate.ts
+++ /@gorhom/bottom-sheet/src/utilities/animate.ts
@@ -3,10 +3,8 @@
   WithTimingConfig,
   withTiming,
   withSpring,
   AnimationCallback,
-  // @ts-ignore
-  ReduceMotion,
 } from 'react-native-reanimated';
 import { ANIMATION_CONFIGS, ANIMATION_METHOD } from '../constants';
 
 interface AnimateParams {
@@ -27,16 +25,8 @@
   if (!configs) {
     configs = ANIMATION_CONFIGS;
   }
 
-  // Users might have an accessibililty setting to reduce motion turned on.
-  // This prevents the animation from running when presenting the sheet, which results in
-  // the bottom sheet not even appearing so we need to override it to ensure the animation runs.
-  if (ReduceMotion) {
-    // @ts-ignore
-    configs.reduceMotion = ReduceMotion.Never;
-  }
-
   // detect animation type
   const type =
     'duration' in configs || 'easing' in configs
       ? ANIMATION_METHOD.TIMING
